src/screens/user-details/user-details-page.tsx

```diff
@@ -1,24 +1,23 @@
 import {useQuery} from '@apollo/client';
 import React from 'react';
-import {View, Text} from 'react-native';
+import {View} from 'react-native';
+import {HeaderComponent} from '../../components/header.component';
 import {GET_USER_DETAIL} from '../../utils/requests';
-import {styleUserDetails} from './user-details-page.styles';
+import {Content, TextContent} from '../../styles/screens.styles';
 
 export const UserDetailsPage = (props: {id: string}) => {
   const {data} = useQuery(GET_USER_DETAIL, {
     variables: {id: props.id},
   });
   return (
     <View>
-      <Text style={styleUserDetails.title}>{data?.user.name}</Text>
-      <View style={styleUserDetails.content}>
-        <Text style={styleUserDetails.text}>E-mail: {data?.user.email}</Text>
-        <Text style={styleUserDetails.text}>Phone: {data?.user.phone}</Text>
-        <Text style={styleUserDetails.text}>
-          Birth Date: {data?.user.birthDate}
-        </Text>
-        <Text style={styleUserDetails.text}>Role: {data?.user.role}</Text>
-      </View>
+      <HeaderComponent title={data?.user.name} />
+      <Content>
+        <TextContent>E-mail: {data?.user.email}</TextContent>
+        <TextContent>Phone: {data?.user.phone}</TextContent>
+        <TextContent>Birth Date: {data?.user.birthDate}</TextContent>
+        <TextContent>Role: {data?.user.role}</TextContent>
+      </Content>
```

Comment:
isso funciona! a única coisa é que enquanto está carregando o `data`, a tela fica algo assim né?
```
E-mail:
Phone:
Birth Date:
Role:
```
pra resolver isso, dá pra fazer algo assim
```suggestion
      {data && (
        <HeaderComponent title={data.user.name} />
        <Content>
          <TextContent>E-mail: {data.user.email}</TextContent>
          <TextContent>Phone: {data.user.phone}</TextContent>
          <TextContent>Birth Date: {data.user.birthDate}</TextContent>
          <TextContent>Role: {data.user.role}</TextContent>
        </Content>
      )}
```